src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,91 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  selectedTodo: Todo | null
+  todos: Todo[];
+  setModalOpened: (modalOpened: boolean) => void;
+  handleDataModal: (Todo: Todo) => void;
+};
 
-    <tbody>
-      <tr data-cy="todo" className="">
```

Comment:
On this line  you should have className 'has-background-info-light' while todo is selected